cookbook-react/src/recipes/collapses/collapse-cmpc/styles.module.scss

```diff
@@ -0,0 +1,72 @@
+@import 'variables/colors';
```

Comment:
If I'm not mistaken there's no transition/animation for when the content collapses and expands, it would be cool to have it